flask/app/error_handler.py

```diff
@@ -1,11 +1,21 @@
-from flask import Blueprint, jsonify, current_app as app
+from flask import Blueprint, json, jsonify, request, current_app as app
 from werkzeug.exceptions import HTTPException
 import traceback
 from sqlalchemy import exc
 from .extensions import db
 
 error_blueprint = Blueprint("error_handler", __name__)
 
+# dump relavant request info into a string
+def get_request_info() -> str:
+    info: str = "\n"
+    info += f"{str(request)}\n"
```

Comment:
I'm not sure if this is helpful because this will be reflected in a later server log line, e.g. `"GET /api HTTP/1.1" 503`